<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastsWindow</class>
 <widget class="QDialog" name="ForecastsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forecast Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="leftAreaWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="forecastTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>180</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="footerAreaWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Defaults...</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="mainAreaWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="resultTabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="generalTab">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Seismicity Forecasts</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QCheckBox" name="checkBox">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Shapiro</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="text">
                  <string>ETAS</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QCheckBox" name="checkBox_3">
                 <property name="text">
                  <string>Ollinger</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Hazard Assessment</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Risk Assessment</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="inputTab">
         <attribute name="title">
          <string>Input</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="seismicity">
         <attribute name="title">
          <string>Models</string>
         </attribute>
         <widget class="QComboBox" name="modelSelectorComboBox">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>20</y>
            <width>251</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Predictated total rate</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>100</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Log Likelihood</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>70</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Model Performance</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Forecast time</string>
          </property>
         </widget>
         <widget class="QLabel" name="scoreTimeLabel">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>120</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLabel" name="predRateLabel">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>120</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Forecast Results</string>
          </property>
         </widget>
         <widget class="QLabel" name="fcTimeLabel">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>100</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLabel" name="scoreLabel">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>100</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="VoxelViewWidget" name="voxelPlot" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>541</width>
            <height>301</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Spatial Seismicity</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="hazard">
         <attribute name="title">
          <string>Hazard</string>
         </attribute>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Hazard Curves</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Summary</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>171</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>OpenQuake Calculation Id</string>
          </property>
         </widget>
         <widget class="QLabel" name="hazCalcIdLabel">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>70</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="CurvePlotWidget" name="hazPlot" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>541</width>
            <height>301</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="risk">
         <attribute name="title">
          <string>Risk</string>
         </attribute>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>171</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>OpenQuake Calculation Id</string>
          </property>
         </widget>
         <widget class="QLabel" name="riskCalcIdLabel">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>70</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Summary</string>
          </property>
         </widget>
         <widget class="QgsMapCanvas" name="mapWidget" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>541</width>
            <height>401</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="statusAreaWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #0099cc;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Forecast due in 15 minutes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="bottomAreaWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: gray;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="SeismicityPlotWidget" name="timeLineWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VoxelViewWidget</class>
   <extends>QWidget</extends>
   <header>ui.views.plots</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurvePlotWidget</class>
   <extends>QWidget</extends>
   <header>ui.views.plots</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapCanvas</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SeismicityPlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>ui.views.plots</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
